scripts/verify-replace.sh

```diff
@@ -16,26 +22,58 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    err_file=$(mktemp ${BASE_REPO_PATH}/$(basename ${repo})-error.XXX)
+    echo "error output file : ${err_file}"
+    std_out_file=$(mktemp ${BASE_REPO_PATH}/$(basename ${repo})-output.XXX)
+    echo "std output file : ${std_out_file}"
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
-    if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+    make pre-verify 2> >(tee ${err_file})
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+        ERROR_REPO_LIST+="$(basename ${repo}) "
+        ERROR_FILE_LIST+="${err_file}  "
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies 2> >(tee ${err_file}) 1> >(tee ${std_out_file})
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+    ERROR_REPO_LIST+="$(basename ${repo}) " 
+    ERROR_FILE_LIST+="${err_file}  "
+    STD_OUT_FILE_LIST+="${std_out_file} "
+    fi
     echo                                                          
     echo =========================================================================================
     echo                                                           
 done
-if [ ${#ERRORLIST[@]} -ne 0 ]; then
+echo                "Summary"
+if [ ${#ERROR_REPO_LIST[@]} -ne 0 ]; then
     echo "Below are the repos with error: "
-    for e in ${ERRORLIST[*]}
+    for error_repo_name in ${ERROR_REPO_LIST[*]}
     do
-        echo "${e}"
+        for error_file_name in ${ERROR_FILE_LIST[*]}
+        do
+            if [[ ${error_file_name} =~ ${error_repo_name} ]]; then 
+                echo                                                          
+                echo =========================================================================================
+                echo 
+                echo                       "${error_repo_name} has the following errors "
+                echo                                                          
+                echo =========================================================================================
+                echo 
```

Comment:
it would make more sense to move the error banner out of the for loop
```suggestion
        echo                                                          
        echo =========================================================================================
        echo 
        echo                       "${error_repo_name} has the following errors "
        echo                                                          
        echo =========================================================================================
        echo 
        for error_file_name in ${ERROR_FILE_LIST[*]}
        do
            if [[ ${error_file_name} =~ ${error_repo_name} ]]; then 
```